### ISSUE TITLE
Other users' status ("last seen / online") disappeared (with the last WhatsAppWeb update)

### ISSUE THREAD

**Issue by imatteobattista26** [tomer8007/whatsapp-web-incognito#79]
Updated my "WhatsAppWeb" Saturday (March 12, 2022):
and, if I hide my "last seen update", the other users' status ("last seen" or "online") disappears and I cant't see It :/
[WAIncognito version: 1.8.5]

Before the latest "WhatsAppWeb update", everything worked perfectly.

Thanks for your service

**Comment by tomer8007** [comment]
Yes, this is a known limitation of multi-device, please see the README. All you can do for now is to exit the beta program

**Comment by imatteobattista26** [comment]
Oh...ok, got it.

First of all: tanks for the reply, Tomer.

Secondly: It seems I can't exit from the "beta program" [already searched & tried some guides] probably because the "multi-device update" is no longer a "beta version" but It ha become a final version "opened to all" (that's why I had to do the latest "WhatsAppWeb update")

**Comment by tomer8007** [comment]
@imatteobattista26 This is unfortunate :-(
I will keep trying to find a workaround for this problem, but it might not be possible.

**Comment by imatteobattista26** [comment]
In any case:
TankYou for your support & work ;)
